lib/version.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+let _ = require('underscore');
+let JuttleAdapters = require('juttle/lib/runtime/adapters');
+let juttleVersion = require('juttle/package.json').version;
+let juttleServiceVersion = require('../package.json').version;
+let juttleJSDPVersion = require('juttle-jsdp/package.json').version;
+
+const BUILT_IN_ADAPTERS = ['file', 'http', 'http_server', 'stdio', 'stochastic'];
+
+function getVersionInfo() {
+    let adapters = {};
+    JuttleAdapters.list().filter((info) => {
+        return !_.contains(BUILT_IN_ADAPTERS, info.adapter);
+    }).forEach((info) => {
+        adapters[`juttle-${info.adapter}-adapter`] = info.version;
```

Comment:
It'd be good to add a comment indicating that this hackery is temporary until a version of juttle containing https://github.com/juttle/juttle/pull/472 is released.